### ISSUE TITLE
ts2fable creates an (almost) empty file

### ISSUE THREAD

**Issue by nozzlegear** [fable-compiler/ts2fable#90]
Hello, I think this is related to #2 but I figured I'd report the problem here. When using ts2fable on the @types definition for React Router - `ts2fable node_modules/@types/react-router/index.d.ts Fable.Import.ReactRouter.fs` - the tool exits cleanly with no warnings, but when I open up the file the only thing that's in there is the following:

```fs
module rec Fable.Import.ReactRouter
open System
open Fable.Core
open Fable.Import.JS
```

Looking at the TypeScript file, it's all a bunch of export statements for other files nested in different folders. I understand that's not supported right now, but the behavior of exiting without any warnings or errors when nothing was converted is confusing if you don't know about #2.

**Comment by ctaggart** [comment]
@nozzlegear, I agree.

**Comment by ctaggart** [comment]
This should be solved by #216. You can try it out with `yarn global add ts2fable@next`. You will need to use the `--export` option.
